tests/acceptance/cli_test.py

```diff
@@ -1473,6 +1473,15 @@ def test_events_json(self):
         assert Counter(e['action'] for e in lines) == {'create': 2, 'start': 2}
 
     def test_events_human_readable(self):
+
+        def has_timestamp(string):
+            str_iso_datetime = ' '.join(string.split(' ')[:2])
+            try:
+                return isinstance(datetime.datetime.strptime(
+                    str_iso_datetime, '%Y-%m-%d %H:%M:%S.%f'), datetime.datetime)
```

Comment:
I don't think you need the `isinstance` here because `strptime` always returns a `datetime`. 

I would just `return datetime.datetime.strptime(...)`